std/net/curl.d

```diff
@@ -2879,7 +2879,7 @@ struct HTTP
     /** The headers read from a successful response.
      *
      */
-    @property string[string] responseHeaders()
+    @property string[string] responseHeaders() return
     {
         return p.headersIn;
```

Comment:
Technically speaking, isn't this fine because you're returning a reference to GC handled memory rather than a reference to the struct itself?